data-es6.js

```diff
@@ -1059,6 +1082,28 @@ exports.tests = [
       },
     },
     {
+      name: 'for-loop iteration scope',
```

Comment:
ah, perhaps it's fixed in chrome 49. it fails in chrome 48 :-)

either way tho, it'd be ideal to a) say "for-in loop" on every for-in loop, and b) include explicit `false`s for every known failure